Monika After Story/game/script-moods.rpy

```diff
@@ -939,3 +939,24 @@ label mas_mood_grateful_gratefulforyou:
     m 3ekbla "I love you, [player]~"
     $ mas_ILY()
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_loved",
+            prompt="...loved.",
+            category=[store.mas_moods.TYPE_BAD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_loved:
+    m 7hub "Sounds about right, [player]!"
+    m 2kubsa "After all, I love you more than anything in the whole world!"
+    m 2dubsb "And I know everyone else loves you very much too."
+    m 4dubsb "Your friends, family..."
+    m 1fubsa "You deserve all this love and affection, [mas_get_player_nickname()]."
+    m 5fubsb "And you can always count on me too, since I will love you forever."
```

Comment:
Dissolve into body pose 5
```suggestion
    show monika 5fubsb at t11 zorder MAS_MONIKA_Z with dissolve_monika
    m 5fubsb "And you can always count on me too, since I will love you forever."
```